libopenage/util/compiler.h

```diff
@@ -8,25 +8,55 @@
  *
  * May contain some platform-dependent code.
  */
+ #include <ciso646>
 
 // pxd: from libcpp.string cimport string
 #include <string>
 
+/**
+ * DLL entry-point decorations.
+ */
+#if defined(_WIN32)
+	#if defined(libopenage_EXPORTS)
+		#define LIBOPENAGE_API __declspec(dllexport)
+	#else
+		#define LIBOPENAGE_API __declspec(dllimport)
+	#endif /* libopenage_EXPORTS */
+#else
+	#define LIBOPENAGE_API
+#endif
+
+#if defined(_MSC_VER)
+#ifndef HAVE_SSIZE_T
+typedef int ssize_t;
```

Comment:
I agree. And I'd like nothing more than a (POSIX?) standard file providing this declaration.
For instance, MSVC claims to provide `SSIZE_T` in `BaseTsd.h`. My problem with that, however, is with a disagreeing [re]definition from `pyconfig.h`. So I went with a non-conflicting definition, and wrapped it in `_MSC_VER` preprocessor guards. To be completely honest, I haven't even tried building this code for x64 yet. That would be another error to deal with.

I can check if we can reduce the usages of `ssize_t`. But, we would still need a definition that agrees with Python's headers, otherwise it could lead to other problems in the build.